### ISSUE TITLE
Full background video overlapping other sections

### ISSUE THREAD

**Issue by ilokhov** [alvarotrigo/pagePiling.js#65]
Thank you for all your work on this great script.

I believe there is a problem when pagePiling is used to display a full background video in one of the sections. To reproduce it, go to the example page (http://alvarotrigo.com/pagePiling/examples/videoBackground.html) and set the browser window to be wide and short (for example 1280x400px). Then scroll to section 2 which does not contain a video: the video from section 1 is overlapping the top of section 2, obstructing a part of this section.

Do you have any idea why this might be happening?

I've witnessed this on different systems/browsers, but I can definitely reproduce it on latest Chrome (44.0.2403.155) on Mac OS X.

As a temporary solution to this problem I use the script's events to fade the video in and out depending on whether the current section contains a video or not.

**Comment by alvarotrigo** [comment]
> Do you have any idea why this might be happening?

I don't know, but that's not a pagePiling problem but a CSS one. The example seems not to be perfect, but not because of pagepling.js itself.

**Comment by ilokhov** [comment]
Thanks for your answer.

Just to follow up on this issue - I noticed that the same thing does not occur with the full screen video background example on fullPage.js. I compared the CSS in the video examples of pagePiling and fullPage.

In the fullPage example the sections have overflow: hidden set on them. This removes the video overlap in the following sections.

I applied the same CSS to my instance of pagePiling and this solved the issue.
